src/components/TodoList.vue

```diff
@@ -1,13 +1,15 @@
 <template>
-  <div class="grid grid-cols text-xl gap-4">
-    <div v-for="(item, index) in props.listItems">
-      <ToDoItemComp @remove-item="removeTodo" :item="item" :index="index" />
+  <div class="flex flex-col space-y-8">
+    <div v-for="(item, index) in props.listItems" class="space-y-8">
```

Comment:
When you are using props in the template you do not have to prefix it with props.{....}